### PR TITLE
Clarify PII info

### PR DESCRIPTION
**Pull request by SteveLasker**

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -917,9 +917,11 @@ The Transparency Service is trusted with the confidentiality of the Signed State
 Some Transparency Services may publish every Transparent Statement in their logs, to facilitate their dissemination and auditing.
 Others may just return Receipts to clients that present Singed Statements for registration, and disclose the ledger only to auditors trusted with the confidentiality of its contents.
 
-A collection of Transparent Statements leaks no information about the contents of other Transparent Statements registered at the Transparency Service.
+A collection of Transparent Statements must not leak information about the contents of other Transparent Statements registered on the Ledger.
 
-Nonetheless, Issuers should carefully review the inclusion of private/confidential materials in their issued Signed Statements; they may for instance remove any PII, or include instead opaque cryptographic commitments, such as hashes.
+Nonetheless, Issuers should carefully review the inclusion of private/confidential materials in their Statements.
+For example, issuers should remove Personally Identifiable Information (PII) as clear text in the statement.
```
Comment (nealmcb) [inline draft-ietf-scitt-architecture.md:923]: There are cases where pii is exactly the information that we want to include and verify in the ledger, for example in genealogical applications. So I would say `issuers should not unnecessarily include` pii as clear text.

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -917,9 +917,11 @@ The Transparency Service is trusted with the confidentiality of the Signed State
 Some Transparency Services may publish every Transparent Statement in their logs, to facilitate their dissemination and auditing.
 Others may just return Receipts to clients that present Singed Statements for registration, and disclose the ledger only to auditors trusted with the confidentiality of its contents.
 
-A collection of Transparent Statements leaks no information about the contents of other Transparent Statements registered at the Transparency Service.
+A collection of Transparent Statements must not leak information about the contents of other Transparent Statements registered on the Ledger.
```
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-architecture.md:920]:
```suggestion
A collection of Transparent Statements must not leak information about the contents of other Transparent Statements registered on the Transparency Service.
```

File: draft-ietf-scitt-architecture.md (hunk 1 above)
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md:923]: We should leave this to a generic statement, like as much as possible PII as clear statements should be avoided (unless the use case demands it).

**Review by yogeshbdeshpande — APPROVED** [review]
Left few comments, kindly absorb and move on.

I have approved it now!

**Review by OR13 — APPROVED** [review]

File: draft-ietf-scitt-architecture.md (hunk 1 above)
Comment (SteveLasker) [reply draft-ietf-scitt-architecture.md:923]: Concerns around the term PII, as its definition is inconsistent across various governing bodies.
Should we:
- use a more generic reference for the user to be aware of the type of confidential materials?
- use PII as a bucket, that can be interpreted by the user, for the scenario?

A payload can contain:
- clear text - transparent
- opaque, as they may be encrypted
- a link to an external service/storage

**Pull request merged**